octobot_trading/exchanges/exchange_config_data.py

```diff
@@ -29,9 +29,15 @@ def __init__(self, exchange_manager):
         self.exchange_manager = exchange_manager
         self.config = exchange_manager.config
 
+        # dict of exchange supported pairs by enabled currencies from self.config
         self.traded_cryptocurrencies = {}
+        # list of exchange supported enabled pairs from self.config
         self.traded_symbol_pairs = []
+        # list of exchange supported pairs from self.config
+        self.existing_symbol_pairs = []
```

Comment:
I would have prefer something like 
```suggestion
        self.config_symbol_pairs = []
```